package.json

```diff
@@ -24,7 +24,7 @@
         "test": "mocha --grep E2E --invert; jshint *.js packages",
         "test:unit": "istanbul cover _mocha -- -R spec --grep E2E --invert",
         "dtslint": "lerna run dtslint",
-        "geth": "geth-dev-assistant --accounts 5 --tag stable",
+        "geth": "geth-dev-assistant --accounts 5 --tag stable --gasLimit 7000000",
```

Comment:
Geth dev launches with a relatively low block gas limit (~5,900,000). This is just to make sure the client is [using the settings](https://github.com/trufflesuite/truffle/blob/develop/scripts/geth.sh#L23) Truffle uses in it's own tests with Geth, which is higher. In practice most of the Truffle contract tests use gas estimation so it might not be necessary, but setting this removes one variable in case things change over there.